app/res/layout/hierarchy_layout.xml

```diff
@@ -46,10 +46,10 @@ the License.
             android:layout_height="wrap_content"
             android:text="@string/jump_to_previous"
             android:padding="12dp"
-            android:textSize="@dimen/text_medium"
-            android:textColor="@color/cc_brand_color"
+            android:textColor="@color/button_primary_color"
             android:background="@color/cc_core_bg"
             style="@style/CommCare.Button.Default"
+            android:textSize="@dimen/text_medium"
             android:layout_weight="1"/>
 
         <Button
```

Comment:
![image (3)](https://github.com/user-attachments/assets/abbf01f2-9fdc-45b1-bda5-1ef903918284)
Because ahmad raised the issue in rounded design at the top thats why i converted into simple button